__tasks__/build-riffraff-bundle.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+const path = require('path');
+const cpy = require('cpy');
+const fs = require('fs');
+const { promisify } = require('util');
+const { warn, log } = require('./log');
+const exec = promisify(require('child_process').execFile);
```

Comment:
we use https://github.com/sindresorhus/execa elsewhere, which near enough gives you this – should be a quick drop in i think?

Reply:
+1